### ISSUE TITLE
Update solution and project to modern version of Visual Studio

### ISSUE THREAD

**Issue by Brett208** [OutpostUniverse/NetFixClient#2]
Current project files are not compatible with VS 2017. VS can attempt to update automatically.

**Comment by Brett208** [comment]
Spent some time today trying to compile NetFixClient. There are some significant problems with the codebase when trying to compile. I have to edit a fair number of project settings and lines of source code. It looks like ForcedExports and NetFixClient are both hardcoded to work with specific file hierarchies. Both in the SVN repository and the location of Outpost 2 on the computer.

This line in particular worries me: `#pragma comment(lib, "../../API/ForcedExports/Lib/ForcedExports")`
I updated it to be `#pragma comment(lib, "Release/ForcedExports.lib")` which fixed an error associated to it. I don't really understand the purpose of this line though.

Ended the day with about 38 LINK errors. Similar to the following 4:

Error	LNK2001	unresolved external symbol "int __cdecl OP2ForcedExport::scr_snprintf(char *,unsigned int,char const *,...)" (?scr_snprintf@OP2ForcedExport@@YAHPADIPBDZZ)	\NetFixClient\OPUNetTransportLayer.obj	1	

Error	LNK2001	unresolved external symbol "class OP2ForcedExport::CConfig OP2ForcedExport::config" (?config@OP2ForcedExport@@3VCConfig@1@A)	OP2Scrip \NetFixClient\OPUNetGameSelectWnd.obj	1	

Error	LNK2001	unresolved external symbol "class OP2ForcedExport::CConfig OP2ForcedExport::config" (?config@OP2ForcedExport@@3VCConfig@1@A)	\NetFixClient\OPUNetTransportLayer.obj	1	

Error	LNK2001	unresolved external symbol "class OP2ForcedExport::TApp OP2ForcedExport::app" (?app@OP2ForcedExport@@3VTApp@1@A)	OP2Script	 \NetFixClient\Main.obj	1

**Comment by DanRStevens** [comment]
The `#pragma comment(lib, "...")` line is equivalent to a linker include setting.

At the time, I wasn't all that familiar with Visual Studio project settings. It was a chore to get things right, had to be redone when project files were updated during Visual Studio upgrades, and I inevitably only did it for specific project configurations by accident, rather than globally. Hence, I tried to just put the configuration setting into the source file. The pragma should probably be moved to a project linker setting.

The errors are linker errors, showing that it's unable to link to the ForcedExport library. Adjusting linker settings so it can find the dependency should get that to work.

Edit: What I haven't addressed here, is how and where the dependency should be. We'll probably setup a Git submodule at some point.
